src/renderer/controllers/prefs-controller.js

```diff
@@ -16,7 +18,8 @@ module.exports = class PrefsController {
       url: 'preferences',
       setup: function (cb) {
         // initialize preferences
-        state.window.title = 'Preferences'
+        state.window.title = new IntlMessageFormat(
```

Comment:
Since this looks like a common line, can we simplify it to

    i18n.format('preferences')